client/components/jetpack/backup-actions-toolbar/index.tsx

```diff
@@ -1,25 +1,31 @@
+import page from '@automattic/calypso-router';
 import { Button } from '@automattic/components';
 import { Tooltip } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
 import { FunctionComponent } from 'react';
-import { backupClonePath } from 'calypso/my-sites/backup/paths';
+import isA8CForAgencies from 'calypso/lib/a8c-for-agencies/is-a8c-for-agencies';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+import { backupClonePath, backupMainPath } from 'calypso/my-sites/backup/paths';
 import { useDispatch, useSelector } from 'calypso/state';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import siteHasBackups from 'calypso/state/rewind/selectors/site-has-backups';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import BackupNowButton from '../backup-now-button';
 import './style.scss';
 
 interface Props {
 	siteId: number;
-	siteSlug: string;
 }
 
-const BackupActionsToolbar: FunctionComponent< Props > = ( { siteId, siteSlug } ) => {
+const BackupActionsToolbar: FunctionComponent< Props > = ( { siteId } ) => {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
-
+	const siteSlug = useSelector( getSelectedSiteSlug ) as string;
 	const hasBackups = useSelector( ( state ) => siteHasBackups( state, siteId ) );
 
+	// Show the "Copy site" button if accessing on Jetpack Cloud or A4A
+	const showCopySiteButton = isJetpackCloud() || isA8CForAgencies();
```

Comment:
This logic was outside this component. I think it is better to use BackupActionsToolbar everywhere, so we don't need to duplicate code. Before we were rendering `BackupNowButton` directly when the user is navigating through Jetpack Cloud.